apps/nft-metadata/src/styles.css

```diff
@@ -1 +1,11 @@
 /* You can add global styles to this file, and also import other style files */
+/* TODO: Remove this once you figure out how to port over color.css from storybook */
+.internal-landing-card-green-bg {
```

Comment:
This style was included in the colors css file in the storybook but I couldn't figure out how to get it imported into the css for the metadata app. The metadata app generated css includes the import but the import goes nowhere. To unblock I added this with a signal to remove once we figure that out